testutil/filefixture/file_fixtures.go

```diff
@@ -35,6 +35,8 @@ var Beta Fixture = Fixture{"Beta",
 	},
 }.defaults()
 
+var Beta_Hash string = "9GYDihlrhHQRNPV10lms35kogosBekjqJVYzTj0O5H-QJYTU7vf0YAgh3XBWKKBC"
```

Comment:
So we could put in an scan round trip inline in the relevant test here, too, it just kinda feels bad.  Maybe the best endgame is to have hashes for fixtures generated automatically at some point for all IO systems?  I'm waiting for personal irritation to boil over enough to embark on that; feel free to tackle.